sources/models/src/lib.rs

```diff
@@ -241,3 +241,13 @@ struct Metadata {
     md: SingleLineString,
     val: toml::Value,
 }
+
+///// Bootstrap Containers
+
+#[model]
+struct BootstrapContainer {
+    source: Url,
+    mode: ValidBootstrapContainerMode,
+    user_data: ValidBase64,
+    essential: bool,
+}
```

Comment:
Without a `superpowered` flag like host containers, how do users control the level of permissions they need?  One use case we know of is using `apiclient apply` during early startup to apply settings from a file or URL, if it's too big for user data or they want to share settings.  They don't need access to modify disks, or anything else superpowered.  Other use cases, like setting up disks, need a lot more access.

Reply:
Some further discussion led to the decision that bootstrap containers will get the minimal extra permissions they need to serve their purpose (ability to set up the filesystem, disks, etc.) but shouldn't have a superpowered flag because it could be abused, even unintentionally, to break the system in obscure ways.